src/main/java/it/gov/pagopa/payhub/activities/activity/debtposition/ValidateDebtPositionActivityImpl.java

```diff
@@ -0,0 +1,152 @@
+        }
+
+        for (PaymentOptionDTO paymentOptionDTO : debtPositionDTO.getPaymentOptions()) {
+            if (CollectionUtils.isEmpty(paymentOptionDTO.getInstallments())) {
+                throw new ValidationException("At least one installment of the debt position is mandatory");
+            }
+            for (InstallmentDTO installmentDTO : paymentOptionDTO.getInstallments()) {
+                validateInstallment(installmentDTO, debtPositionDTO.getDebtPositionTypeOrg());
+                validatePersonData(installmentDTO.getPayer(), debtPositionDTO.getDebtPositionTypeOrg());
+                validateTransfers(installmentDTO.getTransfers());
+            }
+        }
+    }
+
+    private void validateInstallment(InstallmentDTO installmentDTO, DebtPositionTypeOrgDTO debtPositionTypeOrgDTO) {
+        if (StringUtils.isBlank(installmentDTO.getRemittanceInformation())) {
+            throw new ValidationException("Remittance information is mandatory");
+        }
+        if (installmentDTO.getDueDate() != null && installmentDTO.getDueDate().isBefore(LocalDate.now())) {
+            throw new ValidationException("The due date cannot be retroactive");
+        }
+        if (debtPositionTypeOrgDTO.isFlagMandatoryDueDate() && installmentDTO.getDueDate() == null) {
+            throw new ValidationException("The due date is mandatory");
+        }
+        if (installmentDTO.getAmount() == null) {
+            throw new ValidationException("Amount is mandatory");
+        }
+        if (installmentDTO.getAmount() < 0) {
+            throw new ValidationException("Amount is not valid");
+        }
+        if (debtPositionTypeOrgDTO.getAmount() != null && !installmentDTO.getAmount().equals(debtPositionTypeOrgDTO.getAmount())) {
+            throw new ValidationException("Amount is not valid for this debt position type org");
+        }
+    }
+
+    private void validatePersonData(PersonDTO personDTO, DebtPositionTypeOrgDTO debtPositionTypeOrgDTO) {
+        if (personDTO == null) {
+            throw new ValidationException("The debtor is mandatory for installment");
+        }
+
+        String uniqueIdentificationCode = personDTO.getUniqueIdentifierCode();
+
+        if (debtPositionTypeOrgDTO.isFlagAnonymousFiscalCode()) {
+            if (personDTO.isFlagAnonymousIdentifierCode()) {
+                uniqueIdentificationCode = "ANONIMO";
+            } else if (StringUtils.isBlank(personDTO.getUniqueIdentifierCode())) {
+                throw new ValidationException("This organization installment type or installment does not allow an anonymous unique identification code");
+            }
+        } else {
+            if (StringUtils.isBlank(personDTO.getUniqueIdentifierCode())) {
+                throw new ValidationException("Unique identification code is mandatory");
+            }
+        }
+        personDTO.setUniqueIdentifierCode(uniqueIdentificationCode);
```

Comment:
let's just set the value, the uniqueIdentifiedCode is not more used
```suggestion
        if (debtPositionTypeOrgDTO.isFlagAnonymousFiscalCode()) {
            if (personDTO.isFlagAnonymousIdentifierCode()) {
                personDTO.setUniqueIdentifierCode("ANONIMO");
            } else if (StringUtils.isBlank(personDTO.getUniqueIdentifierCode())) {
                throw new ValidationException("This organization installment type or installment does not allow an anonymous unique identification code");
            }
        } else {
            if (StringUtils.isBlank(personDTO.getUniqueIdentifierCode())) {
                throw new ValidationException("Unique identification code is mandatory");
            }
        }
```